build/Linux+BSD/portable/copy-libs

```diff
@@ -79,8 +79,9 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
   #copyLib libQt5WebChannel.so.5  # Not present on Debian, and apparently not needed
-  copyLib libQt5WebKit.so.5
-  copyLib libQt5WebKitWidgets.so.5
+  copyLib libQt5WebEngine.so.5
+  copyLib libQt5WebEngineCore.so.5
+  copyLib libQt5WebEngineWidgets.so.5
```

Comment:
I can't comment on the non Qt libs. For the Qt libs, I think only the 3 webengine one are needed by MuseScore binary but then why would check-depends report them?